village/src/App.js

```diff
@@ -112,8 +113,41 @@ class App extends Component {
           <NavLink to="/smurf-form">Add Smurf</NavLink>
           </div>
         </NavBar>
-        <Route path="/smurf-form" render={props => <SmurfForm handleInputChange={this.handleInputChange} isUpdating={this.state.isUpdating} updateSmurf={this.updateSmurf} addSmurf={this.addSmurf} name={this.state.name} age={this.state.age} height={this.state.height} {...props} />}/>
-        <Route exact path="/" render={props => <Smurfs smurfs={this.state.smurfs} deleteSmurf={this.deleteSmurf} initiateUpdate={this.initiateUpdate} {...props}/>}/>
+        <Route 
+          path="/smurf-form" 
+          render={props => 
+            <SmurfForm 
+              handleInputChange={this.handleInputChange} 
+              isUpdating={this.state.isUpdating} 
+              updateSmurf={this.updateSmurf} 
+              addSmurf={this.addSmurf} 
+              name={this.state.name} 
+              age={this.state.age} 
+              height={this.state.height} 
+              {...props} 
+            />}
+        />
+        <Route 
+          exact 
+          path="/" 
+          render={props => 
+            <Smurfs 
+              smurfs={this.state.smurfs} 
+              deleteSmurf={this.deleteSmurf} 
+              initiateUpdate={this.initiateUpdate} 
+              {...props}
+            />}
+        />
+        <Route 
+          path="/smurf/:id" 
+          render={props => 
+            <SingleSmurf 
+              {...props} 
```

Comment:
Not going to break your code or anything but try to keep your patterns consistent by putting your {...props} spread up top as with the other routes.